scania-digital-design-system/sdds
# feat(stepper): webcomponent

**Describe pull-request**  
Web component version of the stepper component and its children.

**Solving issue**  
Fixes: [AB#2958](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2958)

**How to test**  
1. Go to storybook link below
2. Check in Components -> Stepper -> Web component
3. Try out the controls and see that the design matches the one in figma.

**Suggested test steps**


**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._
